benchmark/Microsoft.IdentityModel.Benchmarks/ValidateTokenAsyncTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.TestUtils;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.IdentityModel.Benchmarks
+{
+    [Config(typeof(AntiVirusFriendlyConfig))]
+    public class ValidateTokenAsyncTests
+    {
+        JsonWebTokenHandler jsonWebTokenHandler;
+        JwtSecurityTokenHandler jwtSecurityTokenHandler;
+        SecurityTokenDescriptor tokenDescriptor;
+        string jsonWebToken;
+        TokenValidationParameters validationParameters;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(Default.PayloadClaims),
```

Comment:
I assume Default.PayloadClaims is a collection of System.Security.Claims.Claim.
We do a lot of work translating into a Dictionary<string,object>[see](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/af9abdb1cad7826bc1ffbc420562781306ebd954/src/Microsoft.IdentityModel.Tokens/TokenUtilities.cs#L43)This as this is another degree of freedom and will make a difference in cpu and memory usage.

A minimum baseline would populate SecurityTokenDescriptor.Claims with a representative set of name/value pairs including "exp", "nbf", "iat". These values will need to be dynamic.

Then set JsonWebTokenHandler.SetDefaultTimesOnTokenCreation = false;